api/inventorypb/agents.proto

```diff
@@ -870,6 +901,56 @@ message ChangeQANPostgreSQLPgStatementsAgentResponse {
   QANPostgreSQLPgStatementsAgent qan_postgresql_pgstatements_agent = 1;
 }
 
+// Add/Change QANPostgreSQLPgStatMonitorAgent
+
+message AddQANPostgreSQLPgStatMonitorAgentRequest {
```

Comment:
we need `disable_query_examples` in this request and we need to pass it to pmm-agent like in slowlog agent.